checker/tests/initialization/NotOnlyInitializedTest.java

```diff
@@ -26,7 +26,7 @@ static void testDeref(NotOnlyInitializedTest o) {
 
     static void testDeref2(@UnderInitialization NotOnlyInitializedTest o) {
         // o is not fully iniatlized, so we cannot dereference its fields
-        // :: error: (dereference.of.nullable)
+        // :: error: (dereference.of.nullable) :: error: (method.invocation.invalid)
```

Comment:
Can you add a comment why `f` and `g` have different expected errors?
The same probably applies to `checker/tests/initialization/FlowFbc.java` above.
Why wasn't this two separate errors before? Should it be two errors?